website/content/docs/auth/github.mdx

```diff
@@ -37,6 +37,13 @@ path, specify `-path=/my-path` in the CLI.
 $ vault login -method=github token="MY_TOKEN"
 ```
 
+It is also possible to use the `VAULT_AUTH_GITHUB_TOKE` environment variable to authenticate to GitHub, such as:
```

Comment:
There's a typo here and below -- this should be `VAULT_AUTH_GITHUB_TOKEN`

Reply:
Thank you, should've re-reviewed myself :bow: